src/ui/src/components/core/content/CoreDataframe/CoreDataframeRow.vue

```diff
@@ -0,0 +1,76 @@
+<script setup lang="ts">
+import { computed, PropType, ref } from "vue";
+import CoreDataframeCell from "./CoreDataframeCell.vue";
+import BaseDropdown from "../../base/BaseDropdown.vue";
+import { ARQUERO_INTERNAL_ID } from "./constants";
+
+const props = defineProps({
+	showIndex: { type: Boolean, required: false },
+	indexText: { type: [Number, String], required: true },
+	columns: { type: Array as PropType<string[]>, required: true },
+	row: { type: Object, required: true },
+	useMarkdown: { type: Boolean, required: false },
+	editable: { type: Boolean, required: false },
+	actions: {
+		type: Object as PropType<Record<string, string>>,
+		required: true,
+	},
+});
+
+defineEmits({
+	change: (columnName: string, id: number, value: unknown) =>
+		typeof columnName === "string" &&
+		typeof id === "string" &&
```

Comment:
are you sure about this one ? The type id is declared as number and the expected type is string ?

Is change : id, same than action: index ? If they are, you should use the same variable name

Reply:
You're right, it's a mistake, it should be number